src/covid_shared/paths.py

```diff
@@ -27,56 +25,77 @@
 VARIANTS_DATA_GISAID = DATA_INTAKE_J_DIR / 'GISAID'
 VARIANTS_DATA = DATA_INTAKE_J_DIR / 'variants'
 
-
-# Shared paths
+##################
+# Executor paths #
+##################
 EXEC_R_SCRIPT_PATH = Path('/share/singularity-images/lbd/shells/singR.sh')
 R_SINGULARITY_IMAGE_PATH = Path('/ihme/singularity-images/lbd/releases/lbd_full_20200128.simg')
 
-RCLONE_CONFIG_PATH = Path('/ihme/covid-19/.config/rclone/rclone.conf')
-
-UNVERSIONED_INPUTS_ROOT = Path('/ihme/covid-19/unversioned-inputs')
-STATIC_DATA_INPUTS_ROOT = Path('/ihme/covid-19/static-data')
-SNAPSHOT_ROOT = Path('/ihme/covid-19/snapshot-data/')
-MODEL_INPUTS_ROOT = Path('/ihme/covid-19/model-inputs/')
-VISIT_VOLUME_ROOT = Path('/ihme/covid-19/visit-volume/')
-MORTALITY_RATIO_ROOT = Path('/ihme/covid-19/mortality-ratio')
-INFECTION_FATALITY_RATIO_ROOT = Path('/ihme/covid-19/infection-fatality-ratio')
-INFECTION_HOSPITALIZATION_RATIO_ROOT = Path('/ihme/covid-19/infection-hospitalization-ratio')
-INFECTION_DETECTION_RATE_ROOT = Path('/ihme/covid-19/infection-detection-rate')
-HOSPITAL_DEATH_RATIO_ROOT = Path('/ihme/covid-19/hospitalization-death-ratio')
-DEATHS_SPLINE_OUTPUT_ROOT = Path('/ihme/covid-19/deaths-outputs/')
-PAST_INFECTIONS_ROOT = Path('/ihme/covid-19/past-infections')
-INFECTIONATOR_OUTPUTS = Path('/ihme/covid-19/seir-inputs')
-SHAPEFILE_ROOT = Path('/ihme/covid-19/shapefiles')
-WEBSCRAPER_ROOT = Path('/ihme/covid-19/webscrape')
-HISTORICAL_MODEL_ROOT = Path('/ihme/covid-19/historical-model')
-AGE_SPECIFIC_RATES_ROOT = Path('/ihme/covid-19/age-specific-rates')
-SYMPTOM_SURVEY_ROOT = Path('/ihme/covid-19/symptom-survey-data')
-
-VARIANT_OUTPUT_ROOT = Path('/ihme/covid-19/variant-scaleup')
-MASK_USE_OUTPUT_ROOT = Path('/ihme/covid-19/mask-use-outputs')
-PNEUMONIA_OUTPUT_ROOT = Path('/ihme/covid-19/pneumonia')
-POPULATION_DENSITY_OUTPUT_ROOT = Path('/ihme/covid-19/population-density')
-MOBILITY_COVARIATES_OUTPUT_ROOT = Path('/ihme/covid-19/mobility-covariate')
+################
+# Shared paths #
+################
+# Concrete share drive root
+COVID_19_2 = Path('/ihme/covid-19-2')
+ARCHIVE_ROOT = COVID_19_2 / 'archive'
+
+# Main share drive root. This is a symlink farm for COVID_19_2.
+# All paths should point here.
+COVID_19 = Path('/ihme/covid-19')
+
+# Shared config for running rclone on the IHME OneDrive
+RCLONE_CONFIG_PATH = COVID_19 / '.config' / 'rclone' / 'rclone.conf'
+
+# Top level directories.  These represent, mostly, outputs of pipeline stages.
+# Keep them in alphabetical order.
+# TODO: This should definitely be in a container.
```

Comment:
@collijk Can you explain this todo?

Reply:
For sure.  These things should live in a custom struct or a generic container with a good name ( a tuple or a dict ).  I don't want to break backwards compatibility though, so it needs some thought.